src/timeline/TimelineModel.cpp

```diff
@@ -867,6 +867,19 @@ TimelineModel::fetchMore(const QModelIndex &)
     setPaginationInProgress(true);
 
     events.fetchMore();
+
+    /*
```

Comment:
This fetchMore actually starts an async computation. The messages are not actually fetched yet in this place.